narps_open/pipelines/team_X19V.py

```diff
@@ -1,611 +1,1014 @@
+            print(e)
+        else:
+            print("The directory was deleted successfully")
+
+    def get_group_level_analysis(
+        self,
+        method,
+    ):
+        """Return the group level of analysis workflow.
+
+        Parameters:
+                - method: one of "equalRange", "equalIndifference" or "groupComp"
+
+
+        Returns:
+                - l3_analysis: Nipype WorkFlow
+        """
+
+        # Infosource Node - To iterate on subject and runs
+        infosource_3rdlevel = Node(
+            IdentityInterface(
+                fields=[
+                    "contrast_id",
+                    "exp_dir",
+                    "result_dir",
+                    "output_dir",
+                    "working_dir",
+                    "subject_list",
+                    "method",
+                ],
+                exp_dir=self.directories.dataset_dir,
+                result_dir=self.directories.results_dir,
+                output_dir=self.directories.output_dir,
+                working_dir=self.directories.working_dir,
+                subject_list=self.subject_list,
+                method=method,
+            ),
+            name="infosource_3rdlevel",
+        )
+        infosource_3rdlevel.iterables = [("contrast_id", self.contrast_list)]
+
+        # Templates to select files node
+        copes_file = join(
+            self.directories.output_dir,
+            "l2_analysis",
+            "_contrast_id_{contrast_id}_subject_id_*",
+            "cope1.nii.gz",
+        )
+
+        varcopes_file = join(
+            self.directories.output_dir,
+            "l2_analysis",
+            "_contrast_id_{contrast_id}_subject_id_*",
+            "varcope1.nii.gz",
+        )
+
+        participants_file = join(self.directories.dataset_dir, "participants.tsv")
+
+        mask_file = join(
+            self.directories.dataset_dir, "..", "NARPS-X19V", "hypo1_unthresh.nii.gz"
+        )
+
+        template = {
+            "cope": copes_file,
+            "varcope": varcopes_file,
+            "participants": participants_file,
+            "mask": mask_file,
+        }
+
+        # SelectFiles node - to select necessary files
+        selectfiles_3rdlevel = Node(
+            SelectFiles(template, base_directory=str(self.directories.results_dir)),
+            name="selectfiles_3rdlevel",
+        )
+
+        datasink_3rdlevel = Node(
+            DataSink(
+                base_directory=str(self.directories.results_dir),
+                container=self.directories.output_dir,
+            ),
+            name="datasink_3rdlevel",
+        )
+
+        merge_copes_3rdlevel = Node(Merge(dimension="t"), name="merge_copes_3rdlevel")
+        merge_varcopes_3rdlevel = Node(
+            Merge(dimension="t"), name="merge_varcopes_3rdlevel"
+        )
+
+        subgroups_contrasts = Node(
+            Function(
+                input_names=["copes", "varcopes", "subject_ids", "participants_file"],
```

Comment:
"subject_ids", "participants_file"

those are not needed anymore: make sure that all Node definition are updated